apps/src/templates/SignInOrAgeDialog.js

```diff
@@ -101,7 +101,11 @@ class SignInOrAgeDialog extends Component {
     }
 
     //Sets cookie to true when anon user is 13+. False otherwise.
-    sessionStorage.setItem(sessionStorageKey, parseInt(value, 10) >= 13);
+    let over13 = parseInt(value, 10) >= 13;
+    sessionStorage.setItem(sessionStorageKey, over13);
+    if (over13) {
+      reload();
```

Comment:
When a logged out user goes to a dance level or project, this dialog is displayed over the dance level (so the level is already loaded behind the dialog), so we somehow need to trigger an update. My naive approach is to just refresh, but we probably don't want that to happen at a wide scale. Any ideas?

Reply:
I seem to remember running into some similar problems to this last year, where we did just go with a reload.

One option could be to just say that even if you're over 13, you won't see the songs the first time (but will when you advance to the next puzzle). This might be good enough (but may not be). We'd need to check with PM.

If not good enough, we need to figure out what all would need to happen to get the songs. I'm not super familiar with the loading/filtering pathway, but I could conceive of a scenario where we always load all the songs, and then just toggle which we show based on some state we have in redux. Then we wouldn't need to do anything, as everything would update appropriately as soon as we set our state.